### PR TITLE
Use beta image for hassfest

### PR DESCRIPTION
**Pull request by ludeeus**
This should speed up the action run time.

**NB!: This can not be added until the next beta is published.**

### CODE REVIEW

File: hassfest/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-        libc-dev \
-        libffi-dev \
-        openssl-dev \
-    \
-    && rm -rf /var/cache/apk/* \
-    \
-    && git clone --depth 1 https://github.com/home-assistant/core.git /core \
-    \
-    && python3 -m pip install --no-cache-dir -e /core \
+    apk add --no-cache bash \
```
Comment (pvizeli) [inline hassfest/Dockerfile:6]: Bash is on that image

Comment (ludeeus) [reply hassfest/Dockerfile]: Nice nice 👍

File: hassfest/Dockerfile [hunk 2]
```diff
@@ -1,31 +1,15 @@
...
-        openssl-dev \
-    \
-    && rm -rf /var/cache/apk/* \
-    \
-    && git clone --depth 1 https://github.com/home-assistant/core.git /core \
-    \
-    && python3 -m pip install --no-cache-dir -e /core \
-    \
-    && chmod +x /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```
Comment (pvizeli) [inline hassfest/Dockerfile:5]: if you modify the git index with +x for this file, you can just copy the file and don't need this layer for permission

Comment (ludeeus) [reply hassfest/Dockerfile]: Was not aware that was a thing, learned something new today 👍

**Pull request merged**